.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add your own name logo created by you at the top of your readme file to present yourself as a brand in the job market :+1: to catch the recruiters' eyes :blush: